testsuite/tests/asmgen/parsecmm.mly

```diff
@@ -143,6 +143,7 @@ fundecl:
     LPAREN FUNCTION fun_name LPAREN params RPAREN sequence RPAREN
       { List.iter (fun (id, ty) -> unbind_ident id) $5;
         {fun_name = $3; fun_args = $5; fun_body = $7; fun_fast = true;
+         fun_fast_compile = true;
          fun_dbg = debuginfo ()} }
```

Comment:
This `fun_fast_compile` field looks like a leftover from the earlier version of this GPR.